src/scopula/core.clj

```diff
@@ -426,3 +426,66 @@
     (->> (repr-scopes-intersecting rs-1 rs-2)
          (map scope-repr-to-str)
          set)))
+
+;; ## Scope aliases
+
+(def scope-alias-regex
+  (re-pattern (str "^[+]" allowed-word)))
+
+(defn is-scope-alias?
+  [scope]
+  (re-matches scope-alias-regex scope))
+
+
+(defn is-scope-aliases-map?
+  [aliases]
+  (every? is-scope-alias? (keys aliases)))
+
+(defn scopes-expand
+  "Given a set of scopes containing scope aliases expand them.
+
+  Scopes aliases will be replaced, so the output of scopes-expand should not contain
+  any scope alias.
+
+  If some scope alias is missing in the scope-aliases-map, scope expand will throw an exception."
+  [scopes aliases]
```

Comment:
Might be bikesheding, but I'd prefer to have the `aliases-mapping` be the first argument of this function. This way we can easily use `partial` to setup a new ready-to-use function in our services that only cares about the scopes.

Reply:
so... Coming from Haskell I generally prefer to pass the meaningful argument at the end, but I learned that in Clojure this is most of the time the opposite. So people prefer to use `->` than `->>`. I think before deciding I need to check the convention used for other functions of this lib first.